src/js/Draw/L.PM.Draw.Circle.js

```diff
@@ -252,13 +330,18 @@ Draw.Circle = Draw.extend({
       radius = center.distanceTo(latlng);
     }
 
-    if (this.options.minRadiusCircle && radius < this.options.minRadiusCircle) {
-      radius = this.options.minRadiusCircle;
-    } else if (
-      this.options.maxRadiusCircle &&
-      radius > this.options.maxRadiusCircle
-    ) {
-      radius = this.options.maxRadiusCircle;
+    if (this.options.radiusEdit) {
```

Comment:
Is always true because `_finishShape()` is only called if `radiusEdit` is true

Reply:
Fixed in latest commit.